packages/wonder-blocks-tooltip/components/tooltip.js

```diff
@@ -175,20 +158,27 @@ export default class Tooltip extends React.Component<Props, State> {
     }
 
     _renderTooltipAnchor(ids?: IIdentifierFactory) {
-        const {forceAnchorFocusivity} = this.props;
+        const {children, forceAnchorFocusivity} = this.props;
+        const {active, anchorElement} = this.state;
+
+        const popperHost =
+            maybeGetPortalMountedModalHostElement(anchorElement) ||
+            document.querySelector("body");
```

Comment:
Wrapping this in a `_getHost` method makes the code a little easier to read and encapsulates any extra logic we might need (not that there should be any).